pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/StudyDao.java

```diff
@@ -27,20 +24,4 @@
     @SqlQuery("select * from study_exit_request where user_id = :userId")
     @RegisterConstructorMapper(StudyExitRequest.class)
     Optional<StudyExitRequest> findExitRequestForUser(@Bind("userId") long userId);
-
-    @SqlQuery(
-            "select lc.language_code_id, lc.iso_language_code from study_language sl"
-            + " inner join language_code lc on sl.language_code_id = lc.language_code_id"
-            + " inner join umbrella_study us on sl.umbrella_study_id = us.umbrella_study_id where us.guid = :studyGuid"
-    )
-    @RegisterConstructorMapper(LanguageDto.class)
-    Set<LanguageDto> findSupportedLanguagesByGuid(@Bind("studyGuid") String studyGuid);
```

Comment:
I was hoping we will add to `StudyDao` instead of creating a new file `StudyLanguageDao`. But it's a little too late now. I removed this query so we don't have two different ways of doing the same thing.